samples/javascript_nodejs/51.cafe-bot/dialogs/whatCanYouDo/resources/whatCanYouDoCard.json

```diff
@@ -51,7 +51,7 @@
             "choices": [
                 {
                     "title": "book a table for 2 in seattle",
-                    "value": " {\"intent\": \"Book_Table\", \"cafeLocation\": \"seattle\", \"number\": \"2\", \"text\": \"book a table for 2 in seattle\"}"
```

Comment:
>cafeLocation [](start = 62, length = 12)

Curious why this was removed?

Reply:
as a sample, I just wanted the dialog to start with asking for location. No other significant reason.